.github/scripts/trymerge.py

```diff
@@ -996,8 +1008,25 @@ def find_matching_merge_rule(pr: GitHubPR,
     """Returns merge rule matching to this pr or raises an exception"""
     changed_files = pr.get_changed_files()
     approved_by = set(pr.get_approved_by())
-    rules = read_merge_rules(repo, pr.org, pr.project)
-    reject_reason = f"PR {pr.pr_num} does not match merge rules"
+    issue_link = get_new_issue_link(
+        org=pr.org,
+        project=pr.project,
+        labels=["module: ci"],
+    )
+    reject_reason = (f"No rules find to match PR, "
+                     f"please [report]{issue_link} this issue to DevX team")
```

Comment:
Don't use `f` decorator in front of string literals. Also, do not rely on the implicit concatenation, but rather add `+` operator or just keep it on the single line (or line length is 120 characters, isn't it?)
```suggestion
    reject_reason = ("No rules find to match PR, "
                     f"please [report]{issue_link} this issue to DevX team")
```